marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -231,40 +231,69 @@ class Jobs {
    * than the Job document that is persisted, which has "job" as its only key.
    * @param flowStep the step as defined in the flow being executed
    * @param stepNumber the number of the step as defined in the flow being executed
+   * @param batchStatus the status of the completed batch
+   * @param items the list of uris updated by the batch execution
+   * @param error the error returned by the batch execution (if any)
+   * @param combinedOptions the options passed to the flow.  Used to determine if processed item hashes should be
+   * added to the batch document.
    * @returns {any}
    */
-  createBatch(job, flowStep, stepNumber) {
+  
+  insertBatch(job, flowStep, stepNumber, batchStatus, items, error, combinedOptions) {
```

Comment:
It shouldn't because we're not updating batch files anymore - they're only getting created once at the end instead of being created at the beginning and updated at the end.